### ISSUE TITLE
godaddy addon domain?

### ISSUE THREAD

**Issue by raj-sidhu** [UeiRicho/cgi-node#24]
HI guys,

Sorry if this is obvious but I have just been introduced to godaddy addon domains.

If I have multiple (shared, addon) domains that are effectively hosted from subdirectories (www.domain1.com being served from www.x.com/domain1) Is the right thing to do follow the instructions to install cgi-node and use one node server for all requests for all addon domains? Or should I have a seperate installation of cgi-node and node for each addon domain? (the latter feels like the right thing to do).

If it is the latter (and I tried) I dont have a bin directory in the addon domain subdirectory under public_html, so please advise.

thanks.

**Comment by raj-sidhu** [comment]
Just fyi this is what I tried. I am trying to add this to an addon domain.

When creating an addon domain you just get a cgi-bin dir within /home/mainDomain/public_html/addonDomain.

I followed the instructions, but used the addon domain as the root. I now have:

bin
cgi-bin
.htaccess
test.jscript

without the .htaccess I see the contents of test.jscript (<? cgiNodeInfo() ?>) but with the .htaccess I get a file not found error. 

The contents of .htaccess are:

Action     cgi-node     /home/aureagle/public_html/porillatest.com/cgi-bin/cgi-node.js 
AddHandler cgi-node     .jscript

Do I need to add anything else because it's in s subdirectory of the main domain?

Permissions are 755 on cgi-bin and node, and 644 on .htaccess. and test.jscript.

alternatively should I be using just one node instance in the main domains root?

tia.
